### DISCUSSION TITLE
How do I remove a plugin?

### DISCUSSION THREAD

**Discussion by themightyoarfish** [LazyVim/LazyVim#785] [Q&A] [answered]
I might be dense, but I can't figure out how to remove a plugin that is default-configured in LazyVim.
On opening nvim, and pressing `l` to the `lazy.nvim` screen, I can press `x` on the offending plugin, and it moves to **Not installed**. But when I quit and reopen nvim, it gets downloaded again. Where can I tell LazyVim not to use a plugin?

**Comment by themightyoarfish** [comment]
I found that I can remove the plugin spec from `~/.local/share/nvim/lazy/LazyVim/lua/lazyvim/plugins/editor.lua`, but is that really the intended way to do it or can I disable a plugin from my own init files?

**Comment by dpetka2001** [comment]
Don't the [docs](https://www.lazyvim.org/configuration/plugins/#-disabling-plugins) answer your question?

**Reply by augustocdias** [reply]
This was removed. How to do it now?

**Reply by folke** [reply]
Updated link: https://www.lazyvim.org/configuration/plugins/#-disabling-plugins

**Reply by washere** [reply]
Thanks, I am using LazyVim and this plugin:

`indent-blankline.nvim`

was causing errors in many config files loading & also other plugins.
Many users, of not just LazyVim but default nvim install & other plugin managers, have complained online about it as it is a persistent bug generator.

So as said in link above, I created the file: 
disabled.lua 
in:
/home/USER/.config/nvim/lua/plugins

(Where USER is your username, I am on LMDE (Debian)).

Then I set this:

`  { "lukas-reineke/indent-blankline.nvim", enabled = false }`

I also had to manually delete his folder, "indent-blankline.nvim" in:

`/home/USER/.local/share/nvim/lazy`

ALSO IF YOU DO NOT THE FULL NAME NEEDED FOR PLUGIN to write in disabled.lua, it can be found in that plugins folder.

Deleted his folder while neovim was not running, so would not auto install as it always did.

It finally stopped it being auto-installed again on I and U (install & update) phases. 

Hope this helps someone using LazyVim or Nvim default or other plugin managers & lands here by search engines, thanks again. :+1:

**Comment by themightyoarfish** [comment, marked as answer]
Well it wasn't obvious to me from that, but it seems you can just make your own plugin file under `lua/plugins/whatever.lua` in which to place the disabling code for other plugins.
